### ISSUE TITLE
Implement Flutter Web accessiblity improvement workaround

### ISSUE THREAD

**Issue by katherineqian** [berkmancenter/frankly#310]
Currently, our Flutter Web app suffers from relatively poor achievement of accessibility standards: see #296 (minimal screen reader support) for an example of an end user reporting issues with the accessibility of our site. While we have already made efforts to improve accessibility from the original state of the product by enabling Semantics in Flutter throughout key parts of the product, there is still much room for improvement in this area as demonstrated by #296. 

Additionally, inability to support pinch-to-zoom is a known issue with Flutter Web that also impacts our product, and causes us to fail to be able to meet known accessibility guidelines like WCAG (see [SC 2.5.1](https://w3.org/TR/WCAG22#pointer-gestures): Pointer Gestures (Level A)). 

An accessibility workaround has been proposed in this thread regarding [pinch-to-zoom](https://github.com/flutter/flutter/issues/97305) support in Flutter Web. In the best case, we would be able to build upon this workaround to meet accessibility guidelines. 

This issue represents the work of implementing the accessibility workaround proposed in that thread, as well as evaluating further opportunities to expand on the workaround for future improvements to accessibility.

**Comment by jlemanski1** [comment]
Hi @katherineqian,

Do you mind if I take a look a this issue? I can implement & test the work around that [I proposed in this issue](https://github.com/flutter/flutter/issues/97305#issuecomment-3865205743).
I have a PR open for the Flutter Engine at the moment to solve this without the work around, but that will likely take some time to be reviewed & scoped..

**Comment by katherineqian** [comment]
Hi @jlemanski1, so sorry for the delayed response here, and cool to hear from you! Definitely feel free to go ahead and take a look at this issue - curious to hear what you find. I'll tag you on it for now!

**Comment by jlemanski1** [comment]
Hi @katherineqian,
I've raised a [PR](https://github.com/berkmancenter/frankly/pull/388) for this issue. Please let me know if you have any questions on the solution or thought process!

I also included some Flutter web accessibility info that might be relevant to the [sub issue](https://github.com/berkmancenter/frankly/issues/296). I didn't do any screen reader testing so I cannot say for sure, but hopefully it helps.
